server/data/nl/danielsjf.txt

```diff
@@ -2871,3 +2871,507 @@ Onze droger is alweer stuk.
+Ik heb iemands leven gered door hem te reanimeren.
+De papieren waren vergeeld door de jaren.
+Een slechte verbranding kan koolstofmonoxide creëren.
+De brandstofpomp was stuk.
+Ze was onoverwinnelijk. 
+Eenmaal bezig is ze niet meer te stoppen.
+Soms voelde hij zich een vreemde in zijn eigen land.
+Hij had het stopbord niet gezien en reed zonder te kijken het kruispunt op.
+Ze verleende voorrang aan de vrachtwagen.
+Hij stak over aan het zebrapad.
+Ze kon nog net de middenberm ontwijken.
+De jager schoot maar miste de fazant.
+Het jachtseizoen is weer begonnen.
+Zijn tegenstander was veel sterker en hij maakte geen schijn van kans.
+Hij kon zijn schoonbroer niet luchten.
+Ze kwam goed overeen met haar schoonbroer.
+Ze probeerde hem een hart onder de riem te steken.
+Zijn schoonzus kwam iedere zondag op bezoek.
+Wacht, ik zal je een diagram tekenen.
+Hij geloofde haar voor geen meter.
+Sinds jaar en dag was hij atheïst.
+Haar ouders hebben haar teveel verwend.
+Hij had slechte bedoelingen.
+Door de sterke windvlagen kwam hij nauwelijks vooruit op zijn fiets.
+Ze is ooit nog de landskampioen geweest.
+Ze heeft een valse lach.
+Er ligt nog een oude krant in de schuif.
+Steek je de haard aan?
+Negen nette neven negeren vier vieze venten.
+We moeten concluderen dat onze pogingen gefaald zijn.
+Er komt een dodelijke asteroïde op ons af.
+Dit betekent het einde van de mensheid.
+Bunkers kunnen ons niet beschermen
+Tsunami’s zullen de kustzones treffen.
+De temperatuur zal enorm toenemen.
+Het team speelde goed samen.
+Hockey is een populaire sport.
+Er is een groot verschil tussen hockey en ijshockey.
+De resultaten waren niet navenant.
+Hij at de pizza beleefd met mes en vork.
+Is een pizza hawaï een echte pizza?
+In het oerwoud ben je enkel op jezelf aangewezen. 
+Ze zeggen dat een duizendtal woorden voldoende is om je in een vreemde taal te behelpen.
+De grens wordt goed bewaakt.
+De veldslag was een smet op het blazoen van de generaal.
+De Titanic is gezonken in negentienhonderdentwaalf.
+Ik heb jaar leren kennen op het internaat.
+Ik ging niet zo graag naar de kostschool. 
+De akkers lagen er verlaten bij.
+Door de lange droogte was een groot deel van de oogst mislukt.
+De graanschuren zitten volledig vol.
+Bij het herlezen heb ik nog een aantal dubbele spaties ontdekt.
+Ze is gevallen op de schaatsbaan.
+Mijn buurvrouw is aan het klagen. 
+Mijn buurman maait altijd het gras op zondag. 
+De honden blaffen in de verte. 
+De weerwolf is enkel actief als het volle maan is.
+Ik krijg betaald op het einde van de maand.
+Het was een moeilijke periode, maar hij is veerkrachtig.
+Hij is altijd gelukkig.
+Zij is altijd goedgehumeurd 's ochtends.
+Hij is altijd positief.
+Zij zegt nooit iets negatief over anderen. 
+Complimenten geven is zijn specialiteit.
+Zij heeft hem opgebeurd.
+De vleermuis vloog door de donkere grot.
+Zijn sjaal zat vast in de roltrap.
+Hij was op tijd in het station, maar stond aan het foute perron.
+Heb je een gom voor mij?
```

Comment:
"een gom" => " een kauwgompje" (BE/NL)

Reply:
Haha, het is een gum in NL blijkbaar. De meeste dingen in NL ken ik wel maar van dit verschil had ik toch nog nooit gehoord.